<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_data_for_chart</class>
 <widget class="QDialog" name="user_data_for_chart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Получение данных для построения графика</string>
  </property>
  <widget class="QDialogButtonBox" name="dialog_btns">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="title_plotting">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>361</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Укажите, в каком диапазоне должны &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;располагаться данные для построения &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;графика (порядок ввода не имеет значения):&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_first_result">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Номер строки первого результата</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_last_result">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Номер строки последнего результата</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_result_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; text-decoration: underline;&quot;&gt;Первый результат:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_result_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; text-decoration: underline;&quot;&gt;Последний результат:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialog_btns</sender>
   <signal>accepted()</signal>
   <receiver>user_data_for_chart</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialog_btns</sender>
   <signal>rejected()</signal>
   <receiver>user_data_for_chart</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
